src/ONYXKEYS.ts

```diff
@@ -563,6 +563,9 @@ const ONYXKEYS = {
         /** Expensify cards settings */
         PRIVATE_EXPENSIFY_CARD_SETTINGS: 'private_expensifyCardSettings_',
 
+        /** Expensify cards bank account settings */
+        PRIVATE_EXPENSIFY_CARD_BANK_ACCOUNT_SETTINGS: 'private_expensifyCardBankAccountSettings_',
```

Comment:
This key doesn't need to follow this naming convention because it doesn't exist on the BE. So it might confuse other engineers who might think this is directly co-related to a NVP that we store in the BE. Thoughts? Do we have other Onyx keys that are solely used on the FE?

```suggestion
        /** Expensify cards bank account settings */
        EXPENSIFY_CARD_BANK_ACCOUNT_SETTINGS: 'expensifyCardBankAccountSettings_',
```

Reply:
Make sense.

These 3 are FE only keys.
https://github.com/Expensify/App/blob/1303cc07d5516a12b3ae77386e215da95d745b95/src/ONYXKEYS.ts#L517-L521
https://github.com/Expensify/App/blob/1303cc07d5516a12b3ae77386e215da95d745b95/src/ONYXKEYS.ts#L496

I think EXPENSIFY_CARD_BANK_ACCOUNT_SETTINGS looks good.

Reply:
@MariaHCD updated.